news/2024/2024-12-14-osu-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,583 @@
+| ::{ flag=RU }:: Russian Federation vs. ::{ flag=CN }:: China | Round of 16 | [Text summary](https://osu.ppy.sh/home/news/2024-11-14-osu-world-cup-2024-round-of-16-recap#russian-federation-vs.--china), [VOD](https://www.twitch.tv/videos/2298021032), [video recap](https://www.youtube.com/watch?v=Q3t9MojGc18&t=136s) |
+| ::{ flag=ES }:: Spain vs. ::{ flag=FI }:: Finland | Round of 16 | [Text summary](https://osu.ppy.sh/home/news/2024-11-14-osu-world-cup-2024-round-of-16-recap#spain-vs.--finland), [VOD](https://www.twitch.tv/videos/2298526307), [video recap](https://www.youtube.com/watch?v=Q3t9MojGc18&t=1132s) |
+| ::{ flag=PH }:: Philippines vs. ::{ flag=UA }:: Ukraine | Quarterfinals | [Text summary](https://osu.ppy.sh/home/news/2024-11-20-osu-world-cup-2024-quarterfinals-recap#philippines-vs.--ukraine), [VOD](https://www.twitch.tv/videos/2304248553), [video recap](https://www.youtube.com/watch?v=cld2spAZQ-M&t=792s) |
+| ::{ flag=AU }:: Australia vs. ::{ flag=US }:: United States | Semifinals | [Text summary](https://osu.ppy.sh/home/news/2024-11-29-osu-world-cup-2024-semifinals-recap#australia-vs.--united-states), [VOD](https://www.twitch.tv/videos/2309726416), [video recap](https://www.youtube.com/watch?v=udSXyASuYI4&t=1021s) |
+| ::{ flag=CA }:: Canada vs. ::{ flag=GB }:: United Kingdom | Semifinals | [Text summary](https://osu.ppy.sh/home/news/2024-11-29-osu-world-cup-2024-semifinals-recap#canada-vs.--united-kingdom), [VOD](https://www.twitch.tv/videos/2310083959), [video recap](https://www.youtube.com/watch?v=udSXyASuYI4&t=393s) |
+| ::{ flag=AU }:: Australia vs. ::{ flag=DE }:: Germany | Finals | [Text summary](https://osu.ppy.sh/home/news/2024-12-05-osu-world-cup-2024-finals-recap#australia-vs.--germany), [VOD](https://www.twitch.tv/videos/2315679372), [video recap](https://www.youtube.com/watch?v=-H-rOKG-2C4&t=436s) |
+| ::{ flag=KR }:: South Korea vs. ::{ flag=US }:: United States | Finals | [Text summary](https://osu.ppy.sh/home/news/2024-12-05-osu-world-cup-2024-finals-recap#south-korea-vs.--united-states), [VOD](https://www.twitch.tv/videos/2315417594), [video recap](https://www.youtube.com/watch?v=-H-rOKG-2C4&t=645s) |
+
+## osu! originals
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+Alongside all the high-quality production and entertainment, the 2024 osu! World Cup also featured **13** brand-new [osu! originals](/wiki/osu!_originals), spanning across almost all major music genres. From jazz to hardcore, the quality of not only the songs, but also all other assets such as backgrounds, storyboards, videos and everything in between have helped generate hype and provide the community with content of the highest quality.
```

Comment:
```suggestion
Alongside all the high-quality production and entertainment, the 2024 osu! World Cup also featured **13** brand-new [osu! originals](/wiki/osu!_originals), spanning across almost all major music genres. From jazz to hardcore, the quality of not only the songs, but also all visual assets such as backgrounds, storyboards, videos and everything in between have helped generate hype and provide the community with content of the highest quality.
```

...assuming it's all visuals. this leaves me wondering what "other" means beyond what's listed lol

Reply:
as in graphic design, mapping and perhaps other minor things like sound effects or lyrics (where "everything in between" can cover it all)